src/neo/Wallets/NEP6/NEP6Wallet.cs

```diff
@@ -182,6 +182,20 @@ public override bool DeleteAccount(UInt160 scriptHash)
             }
         }
 
+        public WalletAccount GetAccount()
```

Comment:
I think we don't need `IsDefault` now. It is used in the old UTXO system.

Reply:
Yes and no, I'd name this method `GetDefaultAccount` and as such it has its own use cases. A wallet can contain multiple addresses/keys and when using CLI I don't necessarily want to always add address to all commands, thus I can rely on default address being there and being used when no address is specified.

Reply:
We can change the name as @roman-khimov said, but I think that it could be useful use the default one in certain cases.

Reply:
Renamed.